build/root/WORKSPACE

```diff
@@ -21,16 +21,16 @@ http_archive(
 
 http_archive(
     name = "io_bazel_rules_go",
-    sha256 = "6776d68ebb897625dead17ae510eac3d5f6342367327875210df44dbe2aeeb19",
-    urls = mirror("https://github.com/bazelbuild/rules_go/releases/download/0.17.1/rules_go-0.17.1.tar.gz"),
+    sha256 = "8a0a7fba4e81a7e4a0ae707f3eeeeb05c92e100b4dffbd8b9fd410e1789f1c0a",
```

Comment:
```
[dims@dims-mac 21:41] ~/Downloads ⟩ sha256sum rules_go-0.17.2.tar.gz
6433336b4c5feb54e2f45df4c1c84ea4385b2dc0b6f274ec2cd5d745045eae1f  rules_go-0.17.2.tar.gz
```

Reply:
oops... for anyone wondering, `8a0a7fba4e81a7e4a0ae707f3eeeeb05c92e100b4dffbd8b9fd410e1789f1c0a` is the sha256 of the github download redirect html page